src/config.h

```diff
@@ -30,6 +30,18 @@
 #ifndef __CONFIG_H
 #define __CONFIG_H
 
+struct rewriteConfigState {
+    dict *option_to_line; /* Option -> list of config file lines map */
+    dict *rewritten;      /* Dictionary of already processed options */
+    int numlines;         /* Number of lines in current config */
+    sds *lines;           /* Current lines as an array of sds strings */
+    int needs_signature;  /* True if we need to append the rewrite
+                             signature. */
+    int force_write;      /* True if we want all keywords to be force
+                             written. Currently only used for testing
+                             and debug information. */
+};
```

Comment:
Why add this definition in this PR? I do not understand

Reply:
This was originally in config.c. i needed to move it out to the header file so that I can access that struct from sentinel.c